src/runtime-rs/crates/resource/src/cpu_mem/initial_size.rs

```diff
@@ -168,10 +168,15 @@ fn get_nr_vcpu(resource: &LinuxContainerCpuResources) -> u32 {
 
 fn convert_memory_to_mb(memory_in_byte: i64) -> u32 {
     if memory_in_byte < 0 {
-        0
-    } else {
-        (memory_in_byte / 1024 / 1024) as u32
+        return 0;
+    }
+
```

Comment:
Was this line redundant?

Reply:
I think it improves readability, but I ended up removing. :-)

Thanks, @lifupan!